### DISCUSSION TITLE
Google satellite layer overlapped on QField

### DISCUSSION THREAD

**Discussion by cdavila** [opengisch/QField#4441] [Q&A]
I have a project which includes an XYZ layer for Google Satellite Imagery (URL http://www.google.cn/maps/vt?lyrs=s@189&gl=cn&x={x}&y={y}&z={z}). This layer is correctly displayed on QGIS Desktop, but when I enable it on QField it is mostly covered by map canvas. It is displayed during a fraction of second, but then it is covered by a white layer in all screen but a narrow stripe at the bottom. I have other XYZ layers in the same project that are correctly displayed. It has happened for more than two years now, but it still happens with current version of QField (2.8.3). I also tried recreating project from scratch, but nothing changed.

**Comment by Oisin-JK** [comment]
This also same upo on another forum, but I don't think its been resolved. Can you post an example QField Project with this problem to assist in finding the issue?

**Reply by cdavila** [reply]
[nitratos.zip](https://github.com/opengisch/QField/files/12017478/nitratos.zip)

**Comment by nirvn** [comment]
@cdavila , your XYZ URL actually leads to a redirect, which in turn leads to a QGIS issue handling tile caching.

Short-term fix, use this URL instead: http://www.google.com/maps/vt?lyrs=s@189&gl=cn&x={x}&y={y}&z={z}

Longer-term fix: https://github.com/qgis/QGIS/pull/53792 -- should be in QField's next release.

**Reply by cdavila** [reply]
Provided URL works like a charm, thank!
